### DISCUSSION TITLE
Welcome to Ninja-Run Discussions!

### DISCUSSION THREAD

**Discussion by sfls-huangzeyuan** [sfls-huangzeyuan/Ninja-Run#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MapMaths** [comment]
我直呼【第41号元素】

**Reply by sfls-huangzeyuan** [reply]
hhh

**Reply by sfls-huangzeyuan** [reply]
铌()

**Reply by sfls-huangzeyuan** [reply]
> 我直呼【第41号元素】

话说有什么新的想法吗。。。最近灵感有点枯竭

**Reply by MapMaths** [reply]
e㎜...一个可以使boss或者玩家持续掉血的火焰弹？一个安在玩家脚底下定时炸弹？一个可以反弹直线运动子弹的盾？或者可以让玩家从平台的侧面走，绕过一部分子弹（好扯）；某些效果可以发生什么奇妙的化学反应（？？？）

P.S.这个游戏我还没玩透，所以有些功能可能已经有了而我却不知道🤔

**Reply by MapMaths** [reply]
我感觉我的灵感更枯竭（

**Reply by sfls-huangzeyuan** [reply]
> e㎜...一个可以使boss或者玩家持续掉血的火焰弹？一个安在玩家脚底下定时炸弹？一个可以反弹直线运动子弹的盾？或者可以让玩家从平台的侧面走，绕过一部分子弹（好扯）；某些效果可以发生什么奇妙的化学反应（？？？）
> 
> P.S.这个游戏我还没玩透，所以有些功能可能已经有了而我却不知道🤔

好主意

**Reply by sfls-huangzeyuan** [reply]
> e㎜...一个可以使boss或者玩家持续掉血的火焰弹？一个安在玩家脚底下定时炸弹？一个可以反弹直线运动子弹的盾？或者可以让玩家从平台的侧面走，绕过一部分子弹（好扯）；某些效果可以发生什么奇妙的化学反应（？？？）
> 
> P.S.这个游戏我还没玩透，所以有些功能可能已经有了而我却不知道🤔

其实持续掉血的毒弹有，然后现在是普通的刺也有

**Reply by sfls-huangzeyuan** [reply]
> e㎜...一个可以使boss或者玩家持续掉血的火焰弹？一个安在玩家脚底下定时炸弹？一个可以反弹直线运动子弹的盾？或者可以让玩家从平台的侧面走，绕过一部分子弹（好扯）；某些效果可以发生什么奇妙的化学反应（？？？）
> 
> P.S.这个游戏我还没玩透，所以有些功能可能已经有了而我却不知道🤔

盾的话有雷盾(就是一个标着雷字样的球），吃了之后碰到雷盾的障碍会消失，但是经验不会

**Reply by sfls-huangzeyuan** [reply]
> e㎜...一个可以使boss或者玩家持续掉血的火焰弹？一个安在玩家脚底下定时炸弹？一个可以反弹直线运动子弹的盾？或者可以让玩家从平台的侧面走，绕过一部分子弹（好扯）；某些效果可以发生什么奇妙的化学反应（？？？）
> 
> P.S.这个游戏我还没玩透，所以有些功能可能已经有了而我却不知道🤔

元素反应:[HERE]https://github.com/sfls-huangzeyuan/Ninja-Run/discussions/4

**Comment by MapMaths** [comment]
话说，MinGW是GCC吗？我之前用过但换系统的时候删了……然后就忘了。我现在用的是Equation的GCC，编译的时候每次都得把所有的cout改成wcout才能显示unicode字符🤯

**Reply by sfls-huangzeyuan** [reply]
GCC

**Reply by sfls-huangzeyuan** [reply]
> 话说，MinGW是GCC吗？我之前用过但换系统的时候删了……然后就忘了。我现在用的是Equation的GCC，编译的时候每次都得把所有的cout改成wcout才能显示unicode字符🤯

我给你我用的IDE吧

**Reply by sfls-huangzeyuan** [reply]
https://royqh1979.gitee.io/redpandacpp/

> 话说，MinGW是GCC吗？我之前用过但换系统的时候删了……然后就忘了。我现在用的是Equation的GCC，编译的时候每次都得把所有的cout改成wcout才能显示unicode字符🤯

https://royqh1979.gitee.io/redpandacpp/

**Reply by MapMaths** [reply]
Thanks

**Comment by sfls-huangzeyuan** [comment]
> > e㎜...一个可以使boss或者玩家持续掉血的火焰弹？一个安在玩家脚底下定时炸弹？一个可以反弹直线运动子弹的盾？或者可以让玩家从平台的侧面走，绕过一部分子弹（好扯）；某些效果可以发生什么奇妙的化学反应（？？？）
> > P.S.这个游戏我还没玩透，所以有些功能可能已经有了而我却不知道🤔
> 
> 其实持续掉血的毒弹有，然后现在是普通的刺也有

现在是这样：每次受到蓝色的刺/火球攻击有40%概率会被附加中毒（概率可以改DrugRate）

**Reply by sfls-huangzeyuan** [reply]
@MapMaths
